includes/class-dashboard.php

```diff
@@ -141,14 +141,14 @@ public function register_settings() {
 		// @TODO update option names and locations below this line.
 		// Export page.
 		register_setting( 'press-sync-export', 'ps_options' );
-		register_setting( 'press-sync-export', 'ps_ignore_comments' );
```

Comment:
Why is this removed as part of this ticket?

Reply:
It could easily be moved to its separate issue - I just noticed that that was where a different small bug was coming from. I'll remove it from this PR.